src/Components/ServiceScene/JSONPanel/LogsJsonComponent.tsx

```diff
@@ -243,11 +252,16 @@ const getStyles = (theme: GrafanaTheme2, wrapLogMessage: boolean) => {
       --json-tree-ul-root-padding: ${theme.spacing(3)} 0 ${theme.spacing(2)} 0;
       --json-tree-arrow-left-offset: -${theme.spacing(2)};
       --json-tree-inline: inline-grid;
+      // Scroll offset for sticky header
+      --json-tree-scroll-margin: 26px;
+      // Scroll behavior @todo set "auto" instead of "smooth" for users with prefers-reduced-motion
+      scroll-behavior: smooth;
```

Comment:
```suggestion
      scroll-behavior: ${window.matchMedia('(prefers-reduced-motion: reduce)').matches ? 'auto' : 'smooth'};
```

I'm not a huge fan of @todo comments, as they usually seem to be forgotten.